app/javascript/src/components/GameLogEntry.vue

```diff
@@ -221,20 +235,20 @@ export default {
       return `Imported a total of ${provincesList.length} ${unit} into ${provincesText}.`;
     },
     maneuverAction(payload) {
-      let unit;
-      if (this.board.graph.get(payload.destination).isOcean) {
-        unit = 'a fleet';
-      } else {
-        unit = 'an army';
-      }
+      const unit = this.unitTypeByDestination_Singular(payload.destination);
       const origin = this.capitalize(payload.origin);
       const destination = this.capitalize(payload.destination);
+
       return `Moved ${unit} from ${origin} to ${destination}.`;
     },
     coexistAction(payload) {
+      const units = this.capitalize(this.unitTypeByDestination_Plural(payload.province));
+      // technically it could be a fleet in port sharing the province with an army
```

Comment:
Very good point! But I think using the phrase with `armies` is fine since there would need to be at least one army sharing a province with a fleet for this to come up. Splitting hairs, imo!